portfolio/src/main/webapp/script.js

```diff
@@ -29,3 +29,26 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact; 
 }
+
+/**
+ * Fetches a random message from java servlet and places it into message-container
+ */
+function getRandomMessage() {
+  fetch('/data').then(response => response.json()).then((message) => {
+    const listElement = document.getElementById('message-container');
+    listElement.innerHTML = "";
+    
+    for (i = 0; i < message.length; i++) {
+        listElement.appendChild(
```

Comment:
Consider: 
for (line of message) {
  listElement.appendChild(createListElement(line));
}

Reply:
Actually, it maybe better to use the forEach method of a list object:
message.forEach(element => listElement.appendChild(createListElement(element)));